SETUP.md

```diff
@@ -0,0 +1,54 @@
+# Setup
+
+## Requirements
+
+- Docker
+- Golang 1.19.2+
+
+## Prepare the working directories
+
+The directories are expected to be located in the same folder, as they depend on
+each other. These environment variables are provided to simplify the setup
+process. If you do not want to use them as is, you will need to adjust them
+according to your case.
+
+```bash
+# Export the variable(s).
+export GUACSEC_HOME="${HOME}/projects/guacsec"
```

Comment:
i usually use `go env GOPATH`/src/github.com/gaucsec, not sure if this is still the common convention

Reply:
This was the pre go-module convention. However, as I mentioned in the text above, people can adjust the value to whatever their like. It is mainly to simplify the instructions after  that.